src/browser/ui/dom/components/ReactDOMTextarea.js

```diff
@@ -26,6 +27,11 @@ var invariant = require('invariant');
 var warning = require('warning');
 
 var textarea = ReactElement.createFactory('textarea');
+var hasNoisyInputEvent = false;
+
+if (ExecutionEnvironment.canUseDOM) {
+  hasNoisyInputEvent = 'documentMode' in document && document.documentMode > 9;
```

Comment:
true, though the current way is consistent with similar checks elsewhere in the code base